modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -22,15 +22,16 @@ using RestSharp.Portable.HttpClient;
 using RestSharp;
 {{/netStandard}}
 
-namespace {{packageName}}.Client
-{
+namespace {{packageName}}.Client {
     /// <summary>
     /// API client is mainly responsible for making the HTTP call to the API backend.
     /// </summary>
-    {{>visibility}} partial class ApiClient
-    {
-        private JsonSerializerSettings serializerSettings = new JsonSerializerSettings
-        {
+    {{>visibility}} partial class ApiClient {
+
+        private static Logger log = new Logger(nameof(ApiClient));
+        private const string SERVER_DEFAULT_ADDRESS = "{{basePath}}";
```

Comment:
@iamkarlson He's not saying to get rid of the constant. In mustache, `{{{basePath}}}` with three braces tells the code to use the raw input. `{{basePath}}` with two braces tells mustache to HTML-escape the string, and this can cause issues with URIs.